app/Http/Controllers/Conversation/ConversationController.php

```
@@ -0,0 +1,188 @@
+    {
+        $conversations = auth()->user()
+            ->conversations()
+            ->with(['users', 'messages'])
+            ->get();
+
+        return response()->json(
+            $conversations->map(fn ($conversation) => $this->transformConversation($conversation))
+        );
+    }
+
+    public function show(Conversation $conversation, Request $request): JsonResponse
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)